### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by awong-coatue**
Builds, install, and push forked version of Airflow to ECR - will also display gitsha of deployed version

### CODE REVIEW

File: master-buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+version: 0.2
+
+env:
+  variables:
+    DOCKER_REPO: '069991306234.dkr.ecr.us-east-1.amazonaws.com/datascience/airflow'
```
Comment (awong-coatue) [inline master-buildspec.yml]: Is this fine being public?

Comment (awong-coatue) [reply master-buildspec.yml]: @pathikrit @nzheng-coatue

File: config/airflow.cfg [hunk 2]
```diff
@@ -0,0 +1,609 @@
+[core]
+# The home folder for airflow, default is ~/airflow
```
Comment (pathikrit) [inline config/airflow.cfg]: why are we copying this: https://github.com/coatue/airflow/blob/4ef94ba433ae7275acf9513a8a1ba74435bf55a6/airflow/config_templates/default_airflow.cfg?

Comment (awong-coatue) [reply config/airflow.cfg]: @pathikrit https://github.com/coatue/airflow/blob/4ef94ba433ae7275acf9513a8a1ba74435bf55a6/airflow/config_templates/default_airflow.cfg#L21-L27

Comment (pathikrit) [reply config/airflow.cfg]: My concern is that if we upgrade and new keys get added, we would never know if we do it this way we would never know. Can we simply symlink to that file instead?

Comment (vdat-coatue) [reply config/airflow.cfg]: i recommend not copying this as well but instead adding an additional step to the Dockerfile to concat the default_airflow template and a coatue.cfg file

`cat airflow/airflow/config_templates/default_airflow.cfg coatue.cfg > config/airflow.cfg`

could be as simple as that.

**Comment by awong-coatue** [comment]
@vdat-coatue yeah, the idea would be to rm the dockerfile stuff from the datasyncs repo

File: config/airflow.cfg (hunk 2 above)
Comment (awong-coatue) [reply config/airflow.cfg]: removed `config/airflow.cfg`

**Comment by vdat-coatue** [comment]
LG!

**Pull request merged**